modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUsersAdminManagementToolbarPropsTransformer.js

```diff
@@ -106,21 +86,73 @@ class ManagementToolbarDefaultEventHandler extends DefaultEventHandler {
 			title: dialogTitle,
 			url: accountEntrySelectorURL,
 		});
-	}
+	};
 
-	_updateAccountUsers(url) {
-		const form = this.one('#fm');
+	const selectAccountEntries = (itemData) => {
+		_openAccountEntrySelector(
+			Liferay.Language.get('select'),
+			`${portletNamespace}selectAccountEntries`,
+			Liferay.Language.get(itemData.dialogTitle),
+			itemData.accountEntriesSelectorURL,
+			(selectedItems) => {
+				var redirectURL = Liferay.Util.PortletURL.createPortletURL(
+					itemData.redirectURL,
+					{
+						accountEntriesNavigation: 'accounts',
+						accountEntryIds: selectedItems.value,
+					}
+				);
+
+				window.location.href = redirectURL;
+			}
+		);
+	};
 
-		Liferay.Util.postForm(form, {
+	const _updateAccountUsers = (url) => {
+		const form = document.getElementById(`${portletNamespace}fm`);
+
+		if (!form) {
+			return;
+		}
```

Comment:
Empty line typo.

Reply:
@markocikos I was just curious, what's an empty line typo?

Reply:
There is an empty line missing here 
```js
}
postForm(form, {
```
 I am assuming it's a typo :slightly_smiling_face:

Reply:
Oh ok, :+1:  understood thanks.